app/timetables/models.py

```diff
@@ -106,11 +106,11 @@ class Timetable(SlugifyMixin, TimestampMixin):
     slugify_field = 'name'
 
     def clean(self):
-        # Ensure current_cycle_day and cycle_length are not None before compare
-        if self.current_cycle_day and self.cycle_length:
-            if self.current_cycle_day > self.cycle_length:
+        # Ensure ref_cycle_day and cycle_length are not None before compare
```

Comment:
@mfalade yea I remember. We had a None type error in the PR @andela-kerinoso highlighted. 

![image](https://cloud.githubusercontent.com/assets/10073270/20747221/9e7852d6-b6e9-11e6-8da5-e24251bb7fe5.png)